spacy/training/corpus.py

```diff
@@ -58,6 +58,17 @@ def read_labels(path: Path, *, require: bool = False):
     return srsly.read_json(path)
 
 
+@util.registry.readers("spacy.PlainTextCorpus.v1")
+def create_plain_text_reader(
+    path: Optional[Path],
+    min_length: int = 0,
+    max_length: int = 0,
+) -> Callable[["Language"], Iterable[Doc]]:
+    if path is None:
```

Comment:
Can we add a docstring to the entry point as well? I'm not fond of the duplication myself, but it'll allow for our VSCode extension to display the correct string on hover without having to do anything like parse the function body, locate the class being constructed and pulling the docstring from it.

Reply:
Done.